tests/install/worker_ay_validation.pm

```diff
@@ -0,0 +1,25 @@
+use Mojo::Base 'openQAcoretest';
+use File::Basename;
+use testapi;
+use utils;
+
+sub run {
+    switch_to_root_console;
+    my $profile = '/root/result.xml';
+    install_packages('autoyast2 libxml2-tools');
+    my $template_path = get_var('AUTOYAST');
+    my $profile_template = basename($template_path);
+    assert_script_run "wget $template_path";
+    # run-scripts=true
```

Comment:
What's this? A left-over from something?

Reply:
Actually I forgot about that. the ticket have it. this obviously doesnt work when I run in my machine. I need to test in the OpenQA test

Reply:
Adding the parameter makes the test fails for the reason I added to a comment in the module. VM doesnt have salt installed and validate the script. I am not sure if just installing salt solve the problem